src/browser/api/system.js

```diff
@@ -304,6 +305,19 @@ exports.System = {
             return process.env[varsToExpand] || null;
         }
     },
+    getFocusedWindow: function() {
+        const fWin = electronBrowserWindow.getFocusedWindow();
+        if (fWin) {
+            const win = coreState.getWinById(fWin.id);
+            if (win && win.openfinWindow) {
+                return { 'uuid': win.openfinWindow.uuid, 'name': win.openfinWindow.name };
+            } else {
+                return null;
+            }
+        } else {
+            return null;
+        }
```

Comment:
Can it simply be:

```
const { id } = electronBrowserWindow.getFocusedWindow();
return coreState.getWinObjById(id);
```

or

```
const { id } = electronBrowserWindow.getFocusedWindow();
const { uuid, name } = coreState.getWinObjById(id);
return { uuid, name };
```

Reply:
I tested this code, when calling BrowserWindow.getFocusedWindow() if it returns null, it fails.